styles.css

```diff
@@ -1,730 +1,538 @@
-* {
-  box-sizing: border-box;
+@import "https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap";
+
+/* HTML5 Reset Stylesheets. */
+html,
+body,
+div,
+span,
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+li,
+form,
+article,
+footer,
+header,
+nav,
+section,
+summary,
+main {
   margin: 0;
   padding: 0;
-  color: #344563;
-  list-style: none;
+  border: 0;
+  outline: 0;
+  font-family: Poppins, sans-serif;
+  vertical-align: baseline;
+  background: transparent;
 }
 
-.btn-see-project {
-  color: #396df2;
-  text-decoration: none;
-  letter-spacing: 0.03em;
-  font-weight: 500;
-  font-size: 17px;
-  text-transform: capitalize;
-  border: 1px solid #6070ff;
-  padding: 0.5em;
-  border-radius: 10px;
-  position: relative;
-  margin-left: 1.5rem;
-  margin-bottom: 5rem;
+body {
+  line-height: 1;
 }
 
-/* -----------------nav bar-------- */
-.nav-list {
-  display: flex;
+article,
+footer,
+header,
+nav,
+section,
+img,
+main {
+  display: block;
+}
+
+li {
   list-style: none;
-  align-items: center;
-  justify-content: space-between;
-  margin-right: 0;
-  width: 286px;
-  height: 72px;
 }
 
-.nav a {
-  font-family: Inter, sans-serif;
-  font-style: normal;
-  font-weight: 600;
-  font-size: 15px;
-  line-height: 20px;
-  color: #fff;
+nav ul {
+  list-style: none;
+}
+
+a {
+  margin: 0;
+  padding: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+  background: transparent;
   text-decoration: none;
 }
 
-.nav-list div {
-  margin: 12px 0 12px 24px;
-  font-family: Roboto, sans-serif;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 18px;
-  line-height: 24px;
-  height: 24px;
-  transition-property: margin;
-  transition-duration: 500ms;
+/* HTML5 Reset Stylesheets. */
+
+/* -----Header Section ---- */
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 1rem;
 }
 
 .logo {
-  font-family: Poppins, sans-serif;
-  color: #7f8cff;
-  font-size: 22px;
-  font-weight: 600;
-  line-height: 16px;
-  text-shadow: 2pc;
+  margin: 4px 199px 4px 12px;
+}
+
+.menu {
+  margin: 4px 24px;
 }
 
-/* -----------------nav bar-------- */
+nav {
+  display: none;
+}
 
-/* -------Presentation Section----- */
+/* -----Header Section ---- */
 
-section.main-container {
-  margin-bottom: 2rem;
-  display: grid;
+.wrapper {
```

Comment:
- [ ] I think it's better to fix the styling for the mobile version of the main section. Your portfolio must be amazing for any size of the screen.
<img width="558" alt="Screen Shot 2022-08-27 at 11 11 40 PM" src="https://user-images.githubusercontent.com/47673311/187054534-e320b703-5b67-457c-bc41-932fde0cfa1b.png">